src/SDK.ts

```diff
@@ -318,14 +319,18 @@ export default class SDK implements ISDK {
     const requestHeaders: StringMap = Constants.defaultHeaders;
     addOcUserAgentHeader(this.ocUserAgent, requestHeaders);
 
-    const endpoint = createGetChatTokenEndpoint(currentLiveChatVersion as LiveChatVersion || this.liveChatVersion, authenticatedUserToken ? true : false);
+    const endpoint = createGetChatTokenEndpoint(currentLiveChatVersion as LiveChatVersion || this.liveChatVersion, authenticatedUserToken ? true : false, multiBot);
 
     if (authenticatedUserToken) {
       requestHeaders[OmnichannelHTTPHeaders.authenticatedUserToken] = authenticatedUserToken;
       requestHeaders[OmnichannelHTTPHeaders.authCodeNonce] = this.configuration.authCodeNonce;
     }
 
     let requestPath = `/${endpoint}/${this.omnichannelConfiguration.orgId}/${this.omnichannelConfiguration.widgetId}/${requestId}`;
+    
+    if (MsOcBotApplicationId && MsOcBotApplicationId.length > 0) {
```

Comment:
We have `params` being used which would create those query params. I'd reuse that:

```
if (MsOcBotApplicationId && MsOcBotApplicationId.length > 0) {
  params['Ms-Oc-Bot-Application-Id'] = MsOcBotApplicationId;
}

```